### ISSUE TITLE
[major] fix BIDS insertion script and replace Gender by Sex

### ISSUE THREAD

**Issue by cmadjar** [aces/Loris-MRI#380]
Once 20.2 is merged into major, need to fix the BIDS import script and rename `candidate` table field `Gender` to `Sex` as this field was renamed in major. The code changes on the MRI side was all done to account for the change of field name in the database

**Comment by cmadjar** [comment]
PR was merged so closing the issue
